kqueen_ui/blueprints/ui/templates/ui/cluster_detail.html

```diff
@@ -2,7 +2,12 @@
 
 {% from "ui/partial/form.html" import render_form %}
 
-{% block page_header %}Cluster {{ cluster.name }} detail{% endblock %}
+{% block main_page_header %}
+<div class="page-header">
+  <h2>Cluster {{ cluster.name }} detail</h2>
+  <h5>ID: {{ cluster.id }}</h5>
```

Comment:
Please make its text color lighter

Reply:
![screenshot](https://user-images.githubusercontent.com/4489907/42881480-f9f79bcc-8a9f-11e8-8e69-7fe286b04a44.png)
I think it is thin enough and making it lighter may impair visibility

Reply:
hm, honestly i agree with @ekhomyakova . IMO this color looks lil bit rude. IMO need to do 1 one following things:
1) make it lighter
2)switch color to mirantis-purple
3) summon @b-zone

Reply:
Bogdan says it should be lighter